doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -442,6 +378,15 @@ Induction
    state lemmas without resorting to manually adding these equalities and
    still get enough information in the proofs.
 
+   :n:`{| generalizing | in } {+ @ident }`
+     First generalizes the goal by the given variables so that they are universally
+     quantified in the goal.  This is generally what one wants to do with
+     variables that are inside constructors in the induction hypothesis.  The other
+     ones need not be further generalized.
+
+   There is a long example of :tacn:`dependent induction` and an explanation
+   of the underlying technique :ref:`here <dependent-induction-examples>`.
+
 .. example::
```

Comment:
It would probably make sense to indent this example (and the `.. seealso`) to make them part of the description of `dependent induction`. Same thing above with the examples about `induction`.